<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_newGribproject</class>
 <widget class="QDialog" name="Dlg_newGribproject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Gribouillot project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="projectBrowseBtt">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="projectDirLabel">
     <property name="text">
      <string>Project directory:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="mapImageChkBx">
     <property name="text">
      <string>Copy the map to project folder.</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="boxBtt">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="projectDirEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="mapBrowseBtt">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mapImageEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mapImageLabel">
     <property name="text">
      <string>Map picture file:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="externalPicChckBx">
     <property name="text">
      <string>Automatically copy external pictures to project folder.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectBrowseBtt</tabstop>
  <tabstop>mapBrowseBtt</tabstop>
  <tabstop>mapImageEdit</tabstop>
  <tabstop>projectDirEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>boxBtt</sender>
   <signal>accepted()</signal>
   <receiver>Dlg_newGribproject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxBtt</sender>
   <signal>rejected()</signal>
   <receiver>Dlg_newGribproject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapBrowseBtt</sender>
   <signal>clicked()</signal>
   <receiver>Dlg_newGribproject</receiver>
   <slot>mapImagePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectBrowseBtt</sender>
   <signal>clicked()</signal>
   <receiver>Dlg_newGribproject</receiver>
   <slot>projectDirPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mapImagePath()</slot>
  <slot>projectDirPath()</slot>
 </slots>
</ui>
